dashboard/ui.R

```diff
@@ -0,0 +1,72 @@
+shinyUI(navbarPage(
+  title = "Data Quality Dashboard",
+  theme = "styles.css",
+  tabPanel( #at the top of every page to navigate through the entire dashboard, contains tabs for home
+    title = "Home",
+    navlistPanel(
+       id = "tabset",
+       tabPanel("Summary Data Profile", "Panel one contents"),
+       tabPanel("Info Panel", "Panel two contents")
+           )),
+  tabPanel( #at the top of every page to navigate through the entire dashboard, contains tabs for data quality
+    title = "Data Quality",
+    navlistPanel(
+      id = "tabset",
+      tabPanel("Home", "Panel one contents"),
+      tabPanel("Completeness", "Panel two contents"),
+      tabPanel("Timeliness", "Panel three contents"),
+      tabPanel("Accuracy Scores from SMR Audits",
+               fluidRow(
+                 column(3,
+                        selectInput("SMRaudit", "SMR", choices = c("(All)", unique(hb_mean$Audit)))
+                 ),
+                 column(3, selectInput("Year", "Year", choices = NULL)
+                  ),
+
+                 column(3, selectInput("Healthboard", "Health Board", choices = NULL)
+                  ),
+
+                  column(3, selectInput("DataItemName", "Data Item", choices = NULL)
```

Comment:
To reduce the complexity of this, and maintain usability, it's probably easiest to utilise `shinyWidgets`. Load this package and then use 
```pickerInput("SMRaudit", "SMR", choices = c(unique(hb_mean$Audit)), options = list(`actions-box` = TRUE), multiple = TRUE)``` 
Then the same for the rest.